### ISSUE TITLE
Godot crashes because of cache 

### ISSUE THREAD

**Issue by ababen1** [godotengine/godot#73673]
### Godot version

godot 4 rc2 (and same result on rc1)

### System information

tried on windows 11 and linux 

### Issue description

I've been trying to open my project on my laptop and it just crashes.
If I delete the cache folder (~/.cache/godot on linux or %APPDATA%/Godot on windows), the project opens up all good.
but then after a few seconds it crashes again. I don't even need to interact with the editor, it will just crash. and the cache folder I deleted will be there again.

### Steps to reproduce

I think the issue might be with the custom theme but I'm not sure. I think that because this problem only started when I set a custom theme to my project. and if i go back to the branch without the custom theme it will work normally. 
So basically what I did: set up a custom theme, save it to git, clone the project on another computer, and then the problem started.

### Minimal reproduction project

none

**Comment by anvilfolk** [comment]
An MRP would be absolutely lovely if you could make it happen! Just something where you set a custom theme that causes a crash, zip it up and upload it here.

**Comment by ababen1** [comment]
closing this becuase it works fine in the latest release
